README.tfdoc.hcl

```diff
@@ -217,6 +217,79 @@ section {
             }
           }
         }
+
+        variable "audit_configs" {
+          type = object(audit_log)
```

Comment:
```suggestion
          type = object(audit_config)
```

Reply:
Change made and requested Nathan to regenerate docs